packages/rancher-istio/1.11/rancher-tracing/charts/values.yaml

```diff
@@ -48,3 +48,4 @@ service:
   name: http-query
   type: ClusterIP
   externalPort: 16686
+  
```

Comment:
why did we add this?

Reply:
A new line got added at the end of the file. Will rectify this.